packages/containers/.storybook/config.js

```diff
@@ -49,18 +49,29 @@ api.expression.register('isTrueExpression', (context, first) => {
 });
 
 const modelHasLabelAction = action('modelHasLabel');
-api.expression.register('modelHasLabel', (context) => {
+api.expression.register('modelHasLabel', context => {
 	modelHasLabelAction(context);
 	return !!context.payload.model.label;
 });
 
 function loadStories() {
-	Object.keys(examples).forEach((example) => {
+	Object.keys(examples).forEach(example => {
 		const state = mock.state();
 		state.cmf.settings.views.appheaderbar = {
 			app: 'Hello Test',
 		};
+		state.cmf.settings.views['HeaderBar:default'] = {
+			logo: { name: 'appheaderbar:logo', isFull: true },
+			brand: { name: 'DATA STREAMS'},
+			notification: { name: 'appheaderbar:notification'}
+		};
 		const actions = state.cmf.settings.actions;
+		actions['appheaderbar:logo'] = {
+			icon: 'talend-logo',
+		};
+		actions['appheaderbar:notification'] = {
```

Comment:
Is this impacted by https://github.com/Talend/ui/pull/788 ?

Reply:
yes it is

Reply:
and it has been merged, so should be updated

Reply:
no not action, only views ;)